templates/crontab.erb

```diff
@@ -0,0 +1,25 @@
+################################################################################
```

Comment:
Can you please convert this to an epp template?

Reply:
This change was made to use epp